Sessions/2e-Dictionaries-and-json.ipynb

```diff
@@ -842,44 +767,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": null,
    "id": "0bc2dde4",
    "metadata": {
     "slideshow": {
      "slide_type": "fragment"
     }
    },
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "[('beginning', 1),\n",
-       " ('get', 1),\n",
-       " ('very', 1),\n",
-       " ('tired', 1),\n",
-       " ('bank', 1),\n",
-       " ('having', 1),\n",
-       " ('nothing', 1),\n",
-       " ('do', 1),\n",
-       " ('once', 1),\n",
-       " ('Alice', 2),\n",
-       " ('was', 2),\n",
-       " ('to', 2),\n",
-       " ('and', 2),\n",
-       " ('of', 3),\n",
-       " ('the', 3),\n",
-       " ('or', 3)]"
-      ]
-     },
-     "execution_count": 16,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "sorted(wordcounts.items(),key=lambda x:x[1])"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "98caa654",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_second_item(x):\n",
+    "    return x[1]\n",
+    "get_second_item((2,3))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "235f4fea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorted(wordcounts.items(),key=get_second_item)"
```

Comment:
I don't think it is obvious (with what's been taught so far) what happens here and that these two last blocks do the same, maybe a description would be needed (or make sure you mention it in class).

Reply:
I agree, I think it's important to stress here they can just copy paste? It's difficult to understand but I will mention it in class.

Reply:
Added a docstring to the function, to make things a bit clearer